xbmc/cores/VideoPlayer/DVDClock.cpp

```diff
@@ -100,14 +101,41 @@ double CDVDClock::GetVsyncAdjust()
   return m_vSyncAdjust;
 }
 
+void CDVDClock::Pause(bool pause)
+{
+  CSingleLock lock(m_critSection);
+
+  if (pause && !m_paused)
+  {
+    if (!m_pauseClock)
+      m_speedAfterPause = m_systemFrequency * DVD_PLAYSPEED_NORMAL / m_systemUsed;
+    else
+      m_speedAfterPause = DVD_PLAYSPEED_PAUSE;
+
+    SetSpeed(DVD_PLAYSPEED_PAUSE);
+    m_paused = true;
+  }
+  else if (!pause && m_paused)
+  {
+    SetSpeed(m_speedAfterPause);
+    m_paused = false;
```

Comment:
@popcornmix those 2 statements are in wrong order. SetSpeed won't start the clock in m_paused is true. I will correct it